eclass/mozconfig-v6.60-r1.eclass

```diff
@@ -206,6 +206,8 @@ mozconfig_config() {
 
 	# Stylo is only broken on x86 builds
 	use x86 && mozconfig_annotate 'Upstream bug 1341234' --disable-stylo
+	# Stylo is horribly broken on arm
+	use arm && mozconfig_annotate 'Upstream bug 1341234' --disable-stylo
```

Comment:
Are you sure about this? I was going to drop this even for x86 given that it should be fixed in >=60 according to upstream bug.

Reply:
Yes I'm serious about it. It compiles, but missrenders the whole UI (its the wrong reference bug, though) However, all these commits are work in progress. 

I can't compile from within a qemu-chroot to speed things up, so I can make only one compile and one runtime testing within 24 hrs.